addons/project/models/project.py

```diff
@@ -967,7 +967,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
         search='_search_personal_stage_type_id',
         help="The current user's personal task stage.")
     partner_id = fields.Many2one('res.partner',
-        string='Customer', recursive=True, tracking=True,
+        string='Customer', tracking=True,
```

Comment:
I don't remember why you need to remove the `reverse=True` here. Did you have any issue with that? 🤔

Reply:
@xavierbol Actually because of this change(https://github.com/odoo/enterprise/pull/19812/files#diff-505826809b4301767ac93163dcfc8e5a148d197f2100cf6f1bb7e0aface0681bR60) 
I need to remove recursive from the field. 
As it gives maximum depth recursion error without this change(https://runbot.odoo.com/runbot/build/11569425)

Thank You!

Reply:
Perhaps it is the `partner_id` in the dependency of your compute method the problem.

What do you think? 🤔

Reply:
@xavierbol Done the change.

Reply:
Now, your change here is always necessary? 🤔

Reply:
@xavierbol Without this change branch is getting red. you can see here(https://runbot.odoo.com/runbot/build/11592491)